### DISCUSSION TITLE
Fibers: Alpha Beta, Making them from textures (Goss, Brass, S)

### DISCUSSION THREAD

**Discussion by snayir6** [mtex-toolbox/mtex#1136] [Ask Anything]
Hi @ralfHielscher  and @kilir ,
I have Ni alloys (FCC) and try to produce its fibers from their textures

odf=calcODF(ebsd('Ni-superalloy').orientations,'halfwidth',10*degree);

Alpha fiber is formed by Gos, Brass, P, Rotated goss
1. I define the individual textures components form alpha fiber
```
text_goss=orientation('Euler',[0 45 0]*degree,cs);%%Goss
text_brass=orientation('Euler',[35 45 0]*degree,cs);%%Brass
text_P=orientation('Euler',[70 45 0]*degree,cs);%%P
text_rot_goss=orientation('Euler',[0 45 0]*degree,cs);%%rotated Goss
```

2. I make small fibers with connecting each of the textures
```
f_gb=fibre(text_goss,text_brass);
f_bp=fibre(text_brass,text_P);
f_prg=fibre(text_P,text_rot_goss);
```

3.. Plot them
```
plot(odf,f_gb,'phi1','LineWidth',2,'DisplayName','Goss to Brass');hold on
plot(odf,f_bp,'phi1','LineWidth',2,'DisplayName','Brass to P');hold on
plot(odf,f_prg,'phi1','LineWidth',2,'DisplayName','P to rotated Goss')
legend('show','location','best','FontSize',12);set(gca,'box','off')
```
Mtex output and an ideal solution
 ![mtex_exp](https://user-images.githubusercontent.com/51959635/126200376-7b9a73cb-789b-4b74-913e-348596f1ee11.jpg)
My questions
1. How can I connect these individual fibers (f_gb,f_bp,f_prg) to make a single alpha fiber
2. How can I change x-xis to for example Phi1 

I used Mtex predefined alpha fibers but I want to define them by myself with connecting these individual textures. I pasted Mtex uutput and ideal solution. Would you help me with this
Thanks,
Selda

The previous discussion about connecting beta fibers
https://groups.google.com/g/mtexmail/c/xelkIBWhm7I/m/EVrJDOZ5EgAJ

this is from one of the publications (REF: Microstructure and microtexture evolution of dynamic recrystallization during hot deformation of a nickel-based superalloy, Mingjia Wang a,b, Chaoyang Suna,b,⁎, M.W. Fuc,⁎,)
![image](https://user-images.githubusercontent.com/51959635/126200578-6297f2f0-2aa1-438b-81fa-ebc0a4c91f82.png)

**Comment by ArulUQ** [comment]
Have you tried the command 'add' to join the fibres?
